src/main/java/Nateh.java

```diff
@@ -0,0 +1,107 @@
+import java.util.Scanner;
+
```

Comment:
Saving the linebreak as a constant may reduce code length in the long term, as this will be used repeatedly, I like the idea!